core/domain/suggestion_services.py

```diff
@@ -54,20 +54,9 @@ def create_suggestion(
             accepted/rejected the suggestion.
     """
 
-    # TODO(nithesh): Remove the check for target type once the feedback threads
-    # are generalised for all types of entities. As at the moment feedback
-    # threads can only be linked to explorations, we have this check.
-    # This will be completed as a part of milestone 2 of the generalised review
-    # system project.
-    if target_type == suggestion_models.TARGET_TYPE_EXPLORATION:
-        thread_id = feedback_services.create_thread(
-            target_id, None, author_id, description,
-            DEFAULT_SUGGESTION_THREAD_SUBJECT, has_suggestion=True)
-        # This line and the if..else will be removed after the feedback thread
-        # migration is complete and the IDs for both models match.
-        thread_id = suggestion_models.TARGET_TYPE_EXPLORATION + '.' + thread_id
-    else:
-        raise Exception('Feedback threads can only be linked to explorations')
+    thread_id = feedback_services.create_thread(
```

Comment:
Just to check: in the "old regime" (with the flag not enabled) is this valid?

Reply:
I had to add 'exploration.' at the front for it to work in the old system. Added. Thanks!